From: =?utf-8?q?IOhannes_m_zm=C3=B6lnig?= <umlaeute@debian.org>
Date: Fri, 18 Sep 2020 18:09:23 +0200
Subject: Avoid privacy breach in HTML

Origin: Debian
Forwarded: not-needed
Last-Update: 2024-09-12

Use html5shiv-printshiv from the Debian node-html5shiv package
---
 doc/8.topics/compander-limiter.htm | 2 +-
 doc/8.topics/slop-tilde.htm        | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- puredata.orig/doc/8.topics/compander-limiter.htm
+++ puredata/doc/8.topics/compander-limiter.htm
@@ -12,7 +12,7 @@
       div.column{display: inline-block; vertical-align: top; width: 50%;}
   </style>
   <!--[if lt IE 9]>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
+    <script src="file:///usr/share/nodejs/html5shiv/dist/html5shiv-printshiv.min.js"></script>
   <![endif]-->
 </head>
 <body>
--- puredata.orig/doc/8.topics/slop-tilde.htm
+++ puredata/doc/8.topics/slop-tilde.htm
@@ -12,7 +12,7 @@
       div.column{display: inline-block; vertical-align: top; width: 50%;}
   </style>
   <!--[if lt IE 9]>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
+    <script src="file:///usr/share/nodejs/html5shiv/dist/html5shiv-printshiv.min.js"></script>
   <![endif]-->
 </head>
 <body>
